sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/UnivocityParser.scala

```diff
@@ -234,38 +234,42 @@ private[csv] object UnivocityParser {
    */
   def tokenizeStream(
       inputStream: InputStream,
-      shouldDropHeader: Boolean,
+      dropFirstRecord: Boolean,
```

Comment:
Is this rename related with the change? How about `checkHeader`?

Reply:
renamed it back